portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerModule.java

```diff
@@ -85,6 +83,7 @@ protected void configure() {
     bind(AsymmetricKeyGenerator.class).toInstance(asymmetricKeyGenerator);
     bind(InMemoryDataCopier.class).to(PortabilityInMemoryDataCopier.class);
     bind(ObjectMapper.class).toInstance(context.getTypeManager().getMapper());
+    bind(Layout.class).toInstance(context.getLayout());
```

Comment:
If we're using the EncryptingLayout each time, should just we just keep it out of the context alltogether?

Reply:
We're not going to be, I just haven't settled on a way to inject it from the config yet.

Reply:
Ya I am confused why we need to build this at all...